style.css

```diff
@@ -0,0 +1,466 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+}
+
+html {
+  background-color: lightgray;
+}
+
+body {
+  width: 375px;
+  margin: 0 auto;
+  background-color: rgb(236, 234, 234);
+}
+
+.flex {
+  display: flex;
+}
+
+/* start header part */
+
+.header {
+  height: 50px;
+  background-image: url(Media/header-background-mobile.png.jpg);
+  background-size: cover;
+  background-color: white;
+}
+
+.header .Left_side {
+  width: 50%;
+  height: 100%;
+}
+
+.social-icon a {
+  display: inline-block;
+  width: 18px;
+  height: 18px;
+  text-align: center;
+  line-height: 40px;
+  color: #333;
+}
+
+.header .Left_side a {
+  text-decoration: none;
+}
+
+.header .Left_side .logo {
+  font-size: 18px;
+  font-weight: 700;
+  padding: 8px;
+  color: #6070ff;
+  margin-left: 5px;
+}
+
+.header .Right_side {
+  width: 50%;
+  height: 100%;
+  flex-direction: row;
+  align-content: end;
+}
+
+.header .Right_side div.bar {
+  flex-direction: column;
+  align-items: end;
+  padding: 12px;
+  color: #6070ff;
+  font-size: large;
+}
+
+/* end of header part */
+
+/* start container part */
+
+.container {
+  width: 375px;
+  height: 611px;
+  background-image: url(Media/header-shapes\ mobile.png.jpg);
+  background-size: cover;
+  border-bottom-left-radius: 60px;
+  background-color: white;
+}
+
+.container .me {
+  width: 327px;
+  height: 200px;
+  font-weight: 700;
+  font-size: 40px;
+  padding: 90px 12px 170px;
+  line-height: 52px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.container .me h1 {
+  font-size: 40px;
+}
+
+.container .desc {
+  width: 327px;
+  height: 146px;
+  left: 0;
+  top: 0;
+  padding: 85px 12px 24px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.container .lets_contact {
+  width: 147px;
+  height: 24px;
+  margin-top: 90px;
+  padding-left: 12px;
+}
+
+.container .lets_contact .contact {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 11px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.social_media {
+  padding-top: 0%;
+  margin: 0 auto;
+  display: flex;
+}
+
+.social-icons-set {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.social-icon {
+  list-style: none;
+  margin: 10px;
+}
+
+.social-icon a:hover {
+  color: #6070ff;
+}
+
+.social-icon a i {
+  font-size: 20px;
+}
+
+.social-icon + .social-icon {
+  margin-left: 10px;
+}
+
+/* coming soon */
+
+/* Start middl section  */
+
+.work_section {
+  display: grid;
+}
+
+.middle_section {
+  height: 552px;
+  width: 343px;
+  margin: 100px 16px 0 16px;
+  border-radius: 15px;
+  background-color: white;
+  padding-bottom: 20px;
+}
```

Comment:
- [ ] Well done on applying grid in your section container but here in individual cards, consider using flex to layout elements in the card. by using flex, the alignment of the elements inside cards would get fixed as right now the alignment of the see project button is a bit off.
requirement:
- Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

Reply:
sorry I don not understand this can you explain more?

Reply:
What I am trying to say that you should use `Display: flex` in `.middle_section` as this the class selector of all the works cards.